mlflow/ml-package-versions.yml

```diff
@@ -75,7 +75,7 @@ keras:
 
   models:
     minimum: "3.0.2"
-    maximum: "3.6.0"
+    maximum: "3.7.0"
```

Comment:
Let's revert unrelated changes here. See https://github.com/mlflow/mlflow/pull/13851#discussion_r1860827148 for how to prevent auto-update.